Sources/OAuth2Client/NXOAuth2Request.m

```diff
@@ -171,4 +192,14 @@ - (void)applyParameters:(NSDictionary *)someParameters onRequest:(NSMutableURLRe
     }
 }
 
+- (void)applyHeaders:(NSDictionary *)someHeaders onRequest:(NSMutableURLRequest *)aRequest;
+{
+    for (id key in someHeaders) {
+        id value = someHeaders[key];
+        if ([key isKindOfClass:[NSString class]] && [value isKindOfClass:[NSString class]]) {
```

Comment:
This should be an `NSAssert` since not spacing this right is a programmer error. This fails silently which is dangerous.